src/app/find/locationDetails/locationDetails.css

```diff
@@ -18,24 +18,37 @@
 .categoryCard {
   background-color: #f8f8fc;
   border: 1px solid #dadada !important;
-  padding: 4vw 4vw 4vw 4vw;
+  border-radius: 2px;
   box-shadow: 0vw 2vw 2vw rgba(0,0,0,0.15) !important;
   margin-bottom: 4vh;
   margin-top: 2vh;
 }
 
+.shelterService:last-of-type {
```

Comment:
Nice! Much better to use `:last-of-type` and keep styling logic entirely in CSS files, than the previous way which had the React code keep track of the last service and inline its style in JSX.